pipeline_filtering_node:
  ros__parameters:
    sub_topic: "/gripper_camera/image_raw"
    pub_topic: "/filtered_image"
    filter_params:
      filter_type: "otsu"
      otsu:
        gamma_auto_correction: true
        gamma_auto_correction_weight: 2.0 
        otsu_segmentation: true

# Filter params should reflect the FilterParams struct 
# defined in /include/pipeline_filters/pipeline_processing.hpp